app/src/androidTest/java/com/droidhats/campuscompass/views/ShuttleBusTest.kt

```diff
@@ -159,12 +151,6 @@ class ShuttleBusTest {
                 withContentDescription("SGW TO LOY"),
                 isDisplayed())).perform(click())
 
-        //Checks that a click on the Start Shuttle Trip button is displayed
-        onView(
-            allOf(
-                withId(R.id.navigateWithShuttle), withText("Start Shuttle Trip"),
-                isDisplayed())).perform(click())
-
         //Click to view LOY campus view
         onView(
             allOf(
```

Comment:
From this point on, please remove the checks for the toggle button.
This is causing the test to fail.